DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -958,8 +980,12 @@ function SpawnDatacubes()
         if( dxr.localURL != add_datacubes[i].map ) continue;
 
         loc = add_datacubes[i].location * coords_mult;
-        if( loc.X == 0 && loc.Y == 0 && loc.Z == 0 )
+        if( loc.X == 0 && loc.Y == 0 && loc.Z == 0 ) {
```

Comment:
maybe we should just delete this

Reply:
Maybe, but the warning is actually good, because *presumably* we would always be testing the change after adding datacubes.  That being said, maybe the warning should be a ClientMessage to make sure it's seen?

Reply:
I mean deleting the whole block not just the warning

Reply:
```suggestion
```

Reply:
I did realize that, but I think it's nicer to have the check there *just* to throw the warning, in case someone forgot to add a location.

In theory we could get rid of it selecting a random location, but I don't think there's much harm to that?